File: Example Project Objc/Test/TestCell.m
//
//  TestCell.m
//  Test
//
//  Created by Prince Ugwuh on 8/10/16.
//  Copyright © 2016 TTT. All rights reserved.
//

#import "TestCell.h"

@implementation TestCell

+ (NSString  * _Nonnull)identifier {
    return @"TestCell";
}


- (void)configureItemForReusableView:(NSString *)item {
    self.backgroundColor = [UIColor redColor];
}

@end

File: Example Project Swift/Example Project Swift/TestCell.swift
//
//  TestCell.swift
//  Example Project Swift
//
//  Created by Prince Ugwuh on 10/12/16.
//  Copyright © 2016 MFX Studios. All rights reserved.
//

import UIKit

class TestCell: UICollectionViewCell, CollectionReusableViewDataSource {
    static var identifier: String = "TestCell"
    
    func configure(_ item: Any?) {
        self.backgroundColor = UIColor.red
    }
}
